### ISSUE TITLE
Google Ad Manager and Admob breaks Upgrade All functionality

### ISSUE THREAD

**Issue by kocburak** [AppLovin/AppLovin-MAX-Unity-Plugin#262]
### MAX Plugin Version

5.10.1

### Unity Version

2021.3.20

### Device/Platform Info

Editor

### Current Behavior

If Google Ad Manager and Admob has new versions, Clicking to "Upgrade All" button results with the error below and editor needs to be restarted. If every Upgrade is done manually error does not occur. This happened to me each time there is an google network update.

<img width="528" alt="245196951-8ba5093e-1e6b-458a-9bcf-686eace29e32" src="https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/assets/35556742/1a1ecd96-2715-4b23-a227-bb572d51b359">

### Expected Behavior

Upgrade All button must upgrade all available networks without error.

### How to Reproduce

Set Google Ad Manager and Admob network sdks for older versions.
Open editor.
Click "Upgrade All" button

**Comment by JonathanLiuApp** [comment]
Thank you for bringing this to our attention. Unfortunately, I was unable to reproduce the error. Were you able to reproduce this in our demo app?

**Comment by kocburak** [comment]
Yes I am able to reproduce this issue on demo app. I made a commit to reproduce this issue in [this repo](https://github.com/kocburak/AppLovin-MAX-Unity-Plugin). I opened the project while Android platform on 2021.3.20 is selected if that matters. As you can see from the latest commit I added multiple mediation network that needs to be updated.

**Comment by JonathanLiuApp** [comment]
@kocburak I cloned the example repository you provided, ensuring that all settings were replicated exactly as you had them. However, I was still unable to reproduce the issue you encountered. To assist me further in understanding the problem, could you please provide any console logs you saw when the error occurred?

**Comment by kocburak** [comment]
[Editor.log](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/files/11773711/Editor.log)
Unfortuntely when this windows is displayed whole editor is note reponding. So I can't see any logs. But here is the editor log data. My system is Windows 10 btw. If you need any information about editor settings or any system related information I would be happy to help.

**Comment by kocburak** [comment]
If you need more info, I would be happy to asist.

**Comment by JonathanLiuApp** [comment]
I attempted to reproduce on both Apple and Windows devices, but was still unable to. Could you try force resolving Android with the External Dependency Manager and seeing if the error still occurs? If the error persists, could you provide the versions of the google adapters before you tried to upgrade them?

**Comment by kocburak** [comment]
I does not matter the google adapter version. I force resolved it and still I had it again. Current adapter version on our game is "android 22.1.0.4_ios_10.6.0.1" for admob and  "android 22.1.0.3_ios_10.6.0.1" for Google Ad Manager. Clicked Upgrade All. Then the error showed up again. When I reopened the Editor it updated to  "android 22.1.0.4_ios_10.7.0.0" and  "android 22.1.0.4_ios_10.7.0.0" respectively. It says something like "AndroidManifest.xml.meta system path could't find". My guess is your Upgrade All functionality try to edit same file (AndroidManifest.xml) for both Ad Manager and AdMob but since the edited version didn't reimported yet, it cannot find.
